loom/graphics/gfxVectorGraphics.cpp

```diff
@@ -148,6 +148,7 @@ void VectorGraphics::lineStyle(float thickness, unsigned int color, float alpha,
 void VectorGraphics::textFormat(VectorTextFormat format) {
 	ensureTextFormat();
 	queue->push_back(lmNew(NULL) VectorTextFormatData(lmNew(NULL) VectorTextFormat(format)));
+	currentTextFormat = format;
```

Comment:
So if I clear and then draw some text without setting textFormat() what
would happen?

On Wed, Jul 1, 2015 at 9:44 AM, themuflon <notifications@github.com> wrote:

> In loom/graphics/gfxVectorGraphics.cpp
> <https://github.com/LoomSDK/LoomSDK/pull/333#discussion_r33678997>:
>
> > @@ -148,6 +148,7 @@ void VectorGraphics::lineStyle(float thickness, unsigned int color, float alpha,
> >  void VectorGraphics::textFormat(VectorTextFormat format) {
> >  	ensureTextFormat();
> >  	queue->push_back(lmNew(NULL) VectorTextFormatData(lmNew(NULL) VectorTextFormat(format)));
> > +	currentTextFormat = format;
>
> That would be clear(), but that resets textFormatDirty, which in turn
> also resets the format eventually.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/LoomSDK/LoomSDK/pull/333/files#r33678997>.
>